src/hotspot/share/opto/loopopts.cpp

```diff
@@ -4495,3 +4495,32 @@ void PhaseIdealLoop::move_unordered_reduction_out_of_loop(IdealLoopTree* loop) {
     assert(phi->outcnt() == 1, "accumulator is the only use of phi");
   }
 }
+
+// Clone all nodes in _data_nodes.
+void DataNodeGraph::clone_nodes(Node* new_ctrl) {
```

Comment:
```suggestion
void DataNodeGraph::clone_data_nodes(Node* new_ctrl) {
```
Then the comment would be obsolete